tests/scalers/aws-dynamodb.test.ts

```diff
@@ -0,0 +1,204 @@
+import test from 'ava'
```

Comment:
Could you please comment the whole content of the file, so it is not being executed, we will enable it once we have AWS account in place.